### PR TITLE
AWS distro tutorials for KF-1.4

### PR DESCRIPTION
**Pull request by surajkota**
### Description of your changes:
- Port existing tutorials from v1.3-branch and path changes according to directory structure
- Change examples directory location to be in dist/aws per discussion
- Fixed a few broken links in RDS readme
- [x] update jupyter-web-app location
- [x] update Katib location
- [x] update pipelines location
- [x] update ingress, alb-controller and envoy-filter location
- [x] update the branch checkout to v1.4-branch
- [x] sync common component install locations from generic 1.4 (mainly 1.4 has training operator instead of individual controller like XGboost, TFJob etc.)

### Testing
Manual, WIP
- [x] All links are working including cross readme links
- [x] Images are loading
- [x] Cognito readme
- [x] RDS-S3 reame
- [x] Cognito-RDS-S3 readme

Checked login, ran a sample pipeline and verified rds and s3 connections and created a notebook server with one of DLC image

#27

### CODE REVIEW

**Review by rrrkharse — APPROVED** [review]

File: distributions/aws/examples/rds-s3/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+          dbHost=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
+
+          bucketName=kf-aws-demo-bucket
+          minioServiceHost=s3.amazonaws.com
+          minioServiceRegion=us-west-2
+          ```
+    1. Configure `distributions/aws/apps/pipelines/secret.env` file with your RDS database username and password that were configured when following the steps in Create RDS Instance.
+        - For example, if your username is `admin` and your password is `Kubefl0w` then your `secret.env` file should look like:
+        - ```
+          username=admin
```
Comment (goswamig) [inline distributions/aws/examples/rds-s3/README.md:83]: Can you add an aestrik to change the password if user is using default CFN from step 5 in perquisite.

File: distributions/aws/examples/rds-s3/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    - ```
+      KATIB_MYSQL_DB_DATABASE=kubeflow
+      KATIB_MYSQL_DB_HOST=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
+      KATIB_MYSQL_DB_PORT=3306
+      DB_USER=admin
+      DB_PASSWORD=Kubefl0w
+      ```
+
+
+### 4. Install using the following command:
```
Comment (goswamig) [inline distributions/aws/examples/rds-s3/README.md:108]: Can we say explicitly what are we installing here? 
Is it complete Kubeflow deployment or just RDS + S3 part?

File: distributions/aws/examples/rds-s3/README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+```sh
+while ! kustomize build distributions/aws/examples/rds-s3 | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+```
+
+Once, everything is installed successfully, you can access the Kubeflow Central Dashboard [by logging in to your cluster](../../../../README.md#connect-to-your-kubeflow-cluster).
+
+Congratulations! You can now start experimenting and running your end-to-end ML workflows with Kubeflow.
+
+
+## Uninstall
```
Comment (goswamig) [inline distributions/aws/examples/rds-s3/README.md:119]: same as above comment

**Review by goswamig — COMMENTED** [review]
Minor comments, overall good work. 
This instructions look a lot better.

File: distributions/aws/examples/rds-s3/README.md (hunk 1 above)
Comment (surajkota) [reply distributions/aws/examples/rds-s3/README.md:84]: done

**Review by goswamig — APPROVED** [review]

**Pull request merged**